pages/home/registry-early-access.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: "Arcade Registry Early Access"
+description: "Learn about the Arcade Registry and how to get early access"
+---
+
+import { EarlyAccessRegistrySurvey } from "@/components/EarlyAccessRegistrySurvey";
+
+# The Arcade Registry
+
+The **Arcade Registry** is how agentic tool developers share their work with the world.  Arcade.dev is collecting all the integrations that agents will ever need in one place -  think HuggingFace or Pypi but for LLM tools.  A powerful community of open source and for-profit developers building out robust and evaluated workflows is how elevate agents to being *truly* useful.
```

Comment:
`is how elevate agents to being *truly* useful` --> `is how [we can] elevate agents to being *truly* useful`

Reply:
fixed up in 6f5f11151c036fb60e4e834130acacebb9b6f816